src/twoMean/index.html

```diff
@@ -4,6 +4,7 @@
 ---
 <link rel="stylesheet" href="twoMean.css">
 <div id="two-mean">
+  <h1>Two Means</h1>
```

Comment:
Can we add test coverage?

Reply:
Hi! I'll make sure that this gets test coverage when (if) we get around to doing that.

Reply:
Cool! I won't block this PR then. LGTM